### ISSUE TITLE
Is there an instruction to use this Firefox plugin with an on-premise Language Tool instance?

### ISSUE THREAD

**Issue by 1605200517** [languagetool-org/languagetool-browser-addon#230]
I find Languge Tool awesome, as well as the Firefox browser plugin, but I would like this plugin to communicate with an on-premise/local deployment of LanguageTool API.

- Is this possible?
- If yes, how complicated is this? Is there an instruction (at least rough outline where to start and which steps to follow)?

**Comment by arysin** [comment]
Yes, go to Menu/Addons/Extensions/LanguageTool and change "LanguageTool API Server URL" field to point to your local server/port.

**Comment by danielnaber** [comment]
See https://addons.mozilla.org/firefox/addon/languagetool/ under "Offline use"

**Comment by katrinleinweber** [comment]
Could you please add a note about [the Docker images](https://github.com/languagetool-org/languagetool#docker) there? That's what helped me get LT up & running :-)
